media-libs/libyuv/libyuv-9999.ebuild

```diff
@@ -34,11 +32,22 @@ DEPEND="${RDEPEND}"
 BDEPEND="test? ( dev-cpp/gtest )"
 
 PATCHES=(
-	"${FILESDIR}/0001-fix-install-dirs.patch"
+	"${FILESDIR}/0001-fix-install-dirs-1909.patch"
 	"${FILESDIR}/0002-disable-static-library.patch"
 	"${FILESDIR}/0003-disable-test-tools.patch"
 )
 
+src_unpack() {
+	if [[ ${PV} == "9999" ]] ; then
+		git-r3_src_unpack
+	else
+		mkdir "${P}" || die
```

Comment:
Why not call `default` instead?

Reply:
I had a QA notice about `S=${WORKDIR}` being "deprecated" in cmake eclass.

Reply:
Oh, it unpacks to top-level instead of inside a directory? (In which case, add a comment).

Reply:
> Oh, it unpacks to top-level instead of inside a directory?

Yes, I uploaded the upstream tarball as-is (only change being recompressing it with bz2 instead of gzip) in order make it easier to diff against and update in the future.

> (In which case, add a comment).

Done.

Reply:
> Yes, I uploaded the upstream tarball as-is (only change being recompressing it with bz2 instead of gzip) in order make it easier to diff against and update in the future.

diffoscope can manage that. And with ebuilds pkgdiff-mg and pkgbump would work due to S.